__tests__/data/mp_output.json

```diff
@@ -97,12 +97,16 @@
       "endpoint": "https://api.mixpanel.com/engage/",
       "headers": {},
       "params": {
+<<<<<<< HEAD
```

Comment:
@ItsSudip there are merge conflicts throughout the PR. Seems like you've reused an older branch improperly.